Themes/default/css/index.css

```diff
@@ -3499,11 +3499,26 @@ ul.post_options li {
 #post_draft_options .settings strong {
 	color: #555;
 }
+#post_confirm_buttons {
+	display: flex;
+	flex-direction: row;
+	flex-wrap: wrap;
+	justify-content: space-between;
+	width: 100%;
```

Comment:
would be cleaner to remove this and then drop the `class="floatright"` from the `post_confirm_buttons` element I believe.